LibTrixi.jl/src/api_c.jl

```diff
@@ -177,16 +177,21 @@ trixi_nvariables_cfptr() = @cfunction(trixi_nvariables, Cint, (Cint,))
 
 
 """
-    trixi_get_cell_averages(simstate_handle::Cint)::Cint
+    trixi_load_cell_averages(simstate_handle::Cint)::Cint
 
-Return number of elements (cells)
+Return cell averaged values
+
+Cell averaged values for each cell and each primitive variable are stored in a contiguous
+array, where cell values for the first variable appear first and values for the other
+variables subsequently.
+The given array has to be of correct size and memory has to be allocated beforehand.
```

Comment:
```suggestion
variables subsequently (structure-of-arrays layout).

The given array has to be of correct size and memory has to be allocated beforehand.
```

Reply:
If any docstrings are changed here, they should probably also be changed in the C API docs.